src/modules/flow/iio/iio.json

```diff
@@ -822,6 +822,12 @@
              "name": "sampling_frequency"
          },
          {
+             "data_type": "string",
+             "default": null,
+             "description": "Channel name. If none, try to auto detect channel",
+             "name": "channel_name"
```

Comment:
@laykuanloon @edersondisouza I updated the PR to use the channel_id(as int). For the "string" method @laykuanloon mentioned, let use use another PR to implement it